padigitale-report-generator/src/main/scala/it/pagopa/interop/padigitalereportgenerator/util/ApplicationConfiguration.scala

```diff
@@ -0,0 +1,25 @@
+package it.pagopa.interop.padigitalereportgenerator.util
+
+import com.typesafe.config.{Config, ConfigFactory}
+import it.pagopa.interop.commons.cqrs.model.ReadModelConfig
+
+object ApplicationConfiguration {
+
+  val config: Config = ConfigFactory.load()
+
+  val interfacesContainer: String = config.getString("report-generator.storage.interfaces-container")
```

Comment:
In the case of [this](https://github.com/pagopa/interop-be-jobs/pull/87/files#r1099918513) commit, need to change this config.